lib/chefspec/coverage.rb

```diff
@@ -28,6 +29,8 @@ def #{name}(*args, &block)
     def initialize
       @collection = {}
       @filters    = {}
+      @outputs    = {}
+      add_output(PutsOutput.new)
```

Comment:
```ruby
add_output do |report|
 erb = Erubis::Eruby.new(File.read(@template))
 puts erb.evaluate(report)
end
```

So that covers the default case.